### DISCUSSION TITLE
Welcome to sophgo-sg200x-debian Discussions!

### DISCUSSION THREAD

**Discussion by Fishwaldo** [Fishwaldo/sophgo-sg200x-debian#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by UQuark** [comment]
Hello! I have two projects with Milk-V Duo S in mind.
1. I want to have a USB serial device attached to Duo S and exposed via socat to my home network
2. I want to run klipper on the device to control my 3D printer. I also have an official camera module which I'd like to incorporate to monitor printing process (it's a pretty standard feature for klipper frontends)

**Comment by NowLoadY** [comment]
great works! this can mean a lot!

**Comment by johandevlabs** [comment]
Great work Fishwaldo! I really appreciate this project. 

I am running your Debian image (release 1.4.0) on a Duo S with WiFi and eMMC, and it is a joy to work with.  I wish I was technically proficient in building custom images so I could better support the work you are doing in more meaningful ways.

If it is helpful to anyone, I have collected my notes on Getting Debian set up, interacting with devices over I2C (ADCs, OLED Displays), GPIOs and more here https://github.com/johandevlabs/milk-v-duo-s-notes

One minor thing to note is that version of `duo-pinmux` that Debian image comes with seems to be for Milk V Duo / Duo256. But I could fix this by copying the `duo-pinmux` from the official Milk V duo S buildroot image to my Debian system. 

Thanks agian
